wdl-format/tests/format/seaseq-case/source.formatted.wdl

```diff
@@ -0,0 +1,939 @@
+version 1.0
+
+import "workflows/tasks/bedtools.wdl"
+import "workflows/tasks/bowtie.wdl"
+import "workflows/tasks/fastqc.wdl"
+import "workflows/tasks/macs.wdl"
+import "workflows/tasks/rose.wdl"
+import "workflows/tasks/runspp.wdl"
+import "workflows/tasks/samtools.wdl"
+import "workflows/tasks/seaseq_util.wdl" as util
+import "workflows/tasks/sicer.wdl"
+import "workflows/tasks/sortbed.wdl"
+import "workflows/tasks/sratoolkit.wdl" as sra
+import "workflows/workflows/bamtogff.wdl"
+import "workflows/workflows/mapping.wdl"
+import "workflows/workflows/motifs.wdl"
+import "workflows/workflows/visualization.wdl" as viz
+
+workflow seaseq {
+
+    meta {
+        title: "SEAseq Analysis"
+        summary: "Single-End Antibody Sequencing (SEAseq) Pipeline"
+        description: "A comprehensive automated computational pipeline for all ChIP-Seq/CUT&RUN data analysis."
+        version: "2.0.0"
+        details: {
+            citation: "https://doi.org/10.1186/s12859-022-04588-z",
+            contactEmail: "modupeore.adetunji@stjude.org",
+            contactOrg: "St Jude Children's Research Hospital",
+            contactUrl: "",
+            upstreamLicenses: "MIT",
+            upstreamUrl: "https://github.com/stjude/seaseq",
+            whatsNew: [
+                {
+                    version: "2.0",
+                    changes: [
+                        "version of case/sample only",
+                        "single-end sequencing with input/control sequencing data",
+                        "Initial release",
+                    ],
+                },
+            ],
+        }
+        parameter_group: {
+            reference_genome: {
+                title: "Reference genome",
+                description: "Genome specific files. e.g. reference FASTA, GTF, blacklist, motif databases, FASTA index, bowtie index .",
+                help: "Input reference genome files as defined. If some genome data are missing then analyses using such data will be skipped.",
+            },
+            input_genomic_data: {
+                title: "Input FASTQ data",
+                description: "Genomic input files for experiment.",
+                help: "Input one or more sample data and/or SRA identifiers.",
+            },
+            analysis_parameter: {
+                title: "Analysis parameter",
+                description: "Analysis settings needed for experiment.",
+                help: "Analysis settings; such output analysis file name.",
+            },
+        }
+    }
+
+    parameter_meta {
+        reference: {
+            description: "Reference FASTA file",
+            group: "reference_genome",
+            patterns: [
+                "*.fa",
+                "*.fasta",
+                "*.fa.gz",
+                "*.fasta.gz",
```

Comment:
Should we sort keys like this? Either `patterns` specifically or arrays in `parameter_meta` generally?

Reply:
Similar thinking as above. I don't think that's a good idea, but I'm open to having my mind changed.

Reply:
Fair enough. It just bothers me that this one isn't sorted the same as the arrays below.